buffer_interface/buffer_interface.c

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <stdint.h>
+
+#define BUFFER_SIZE 50
+
+uint32_t buffer1[BUFFER_SIZE] = {0};
+uint32_t buffer2[BUFFER_SIZE] = {0};
+
+enum buff_num {
+    BUFFER1,
+    BUFFER2
+};
+
+typedef enum {
+    false,
+    true
+} bool;
```

Comment:
As part of C99, the bool datatype is available in stdbool.h. You can choose to either keep the current enum or utilize the bool from stdbool.

Reply:
oh thanks for the tip, yeah I should use the lib so do not reinvent the wheel!